### PR TITLE
Feature/dutch translations

### LINKED ISSUE #176: Add Dutch translation
It would be nice to have a Dutch translation of the app. :netherlands: 

The _Translation Editor_ in _Android Studio_ let's you easily identify the missing translations as shown in the screenshot.

Android-Studio-Translation-Editor

Please use the automatic formatting for the string files which Android Studio provides.

- [ ] Add new translations in `app/src/main/res/values-nl` (ignore `placeholder` strings)
  - Adding a new language can be done via the _Translation Editor_: the earth icon at top left
- [ ] Optional: Add new translations in `app/src/ccc35c3/res/values-nl` (becomes visible when you change the _Build Variant_)
- [ ] Optional: Add new translations in `app/src/cccamp2019/res/values-nl` (becomes visible when you change the _Build Variant_)

# Related
- [Localize the UI with Translations Editor](https://developer.android.com/studio/write/translations-editor.html)
- [Contribution guide](https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by spmvanmierlo**
# Description
Contains dutch translations for the app. See issue #176 

# Before
- values-nl/* files did not exist.

# After
- Created the values-nl related files with the dutch translations.

Resolves #176

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    <string name="lecture_list_item_language_removed_content_description">Taal informatie is verwijderd</string>
+    <string name="lecture_list_item_no_video_content_description">Geen video-opname</string>
+    <string name="lecture_list_item_video_content_description">Met video-opname</string>
+    <string name="lecture_list_item_without_video_content_description">Zonder video-opname</string>
+    <string name="placeholder_day_separator_title">Dag 2</string>
+    <string name="placeholder_event_date">12/28/13 5:15 PM - GROTE RUIMTE</string>
+    <string name="placeholder_event_day">Vandaag</string>
+    <string name="placeholder_event_duration">240 min.</string>
+    <string name="placeholder_event_id">ID: 5193</string>
+    <string name="placeholder_event_language">en</string>
```
Comment (johnjohndoe) [inline app/src/main/res/values-nl/strings.xml]: Please don't translate the `placeholder` strings. They are used to render a human friendly preview of the layout files.

**Comment by spmvanmierlo** [comment]
> Almost perfect. One small fix please.
> 
> If possible please also squash your commits into a single commit. Otherwise I can do it later.
> Please update your branch and this pull request via `git push --force origin feature/DutchTranslations`.

Squashed the commits and resolved the placeholder discussion.

**Comment by johnjohndoe** [comment]
@spmvanmierlo Can you reduce the 2 commits into 1 please? You can also edit the resulting commit message.

**Comment by johnjohndoe** [comment]
Happy new year @sjorsvanmierlo! I started managing translations for the EventFahrplan at [Crowdin](https://crowdin.com/project/eventfahrplan). I like to invite you to join there if you like.

**Pull request merged**